src/catalog_provider/glue.rs

```diff
@@ -379,12 +373,21 @@ impl GlueCatalogProvider {
         };
         let format = format_result?;
 
+        let partition_col_names = glue_table
+            .partition_keys()
+            .iter()
+            .map(|c| (c.name().to_owned(), DataType::Utf8))
+            .collect::<Vec<(String, DataType)>>();
+
+        let file_extension = format.file_type().get_ext().clone();
```

Comment:
I did not copy the file_type extension value (because this does not work with compressed files, eg: blah.json.gz would be filtered out in case of compressed json files)